cli/src/config/templates.toml

```diff
@@ -142,6 +133,15 @@ commit_summary_separator = 'label("separator", " | ")'
   time_range.start().ago() ++ label("time", ", lasted ") ++ time_range.duration()'''
 'format_timestamp(timestamp)' = 'timestamp.local().format("%Y-%m-%d %H:%M:%S")'
 
+'format_root_commit(root)' = '''
```

Comment:
nit: `format_log_root` ?

or rename "op_log" aliases to `format_root_operation`/`format_snapshot_operation`?

(I personally prefer the latter)

Reply:
yep, I like the latter too, done